charts/dsp-atlantis/templates/githubCredentials.yaml

```diff
@@ -0,0 +1,25 @@
+{{- /*
+  Credentials for github accounts to be extracted from Vault
+  Primary github account should be that of dsp-atlantis https://github.com/dsp-atlantis
+  Documentation for this api here: https://github.com/tuenti/secrets-manager
+*/}}
+
+apiVersion: secrets-manager.tuenti.io/v1alpha1
+kind: SecretDefinition
+metadata:
+  # name of object as seen in argoCD
+  name: {{ .Values.name }}-github-provider-secretdefinition
+  labels:
+    {{- include "dsp-atlantis.labels" . | nindent 4 }}
+spec:
+  # name: name of the secret in KUBERNETES
+  # keysMap: map of keys in the KUBERNETES secret
+  #   some-name-here: name of the key in the secret in KUBERNETES
+  #     path: path of the secret in VAULT
+  #     name: name of the key in the secret in VAULT 
+  name: atlantis-github-providers
+  keysMap:
+    dsp-atlantis-github-token:
+      path: {{ required "A valid .Values.vault.github.pathPrefix is required" .Values.vault.github.pathPrefix }}/tokens
+      key: ATLANTIS-GITHUB-TOKEN
```

Comment:
Oh, I just checked and the Vault secret is `ATLANTIS_GITHUB_TOKEN` (underscores)

Reply:
fix'd